src/Sentry.Unity.Editor/ConfigurationWindow/OptionsConfigurationTab.cs

```diff
@@ -22,7 +22,7 @@ public static void Display(ScriptableSentryUnityOptions options)
                 "This allows you to change configuration otherwise unavailable from the Editor UI, e.g. set a custom BeforeSend callback." +
                 "\n" +
                 "\n" +
-                "Use 'Buildtime Options Script' in case you need to change build-time behavior, e.g. specify custom Sentry-CLI options " +
+                "Use 'BuildTime Options Script' in case you need to change build-time behavior, e.g. specify custom Sentry-CLI options " +
```

Comment:
```suggestion
                "Use 'Build Time Options Script' in case you need to change build-time behavior, e.g. specify custom Sentry-CLI options " +
```